src/docker/compose_helpers.go

```diff
@@ -17,7 +17,7 @@ func CreateNewContainer(serviceName string, env []string, dockerComposeDir strin
 
 // KillAllContainers kills all the containers
 func KillAllContainers(dockerComposeDir string, logChannel chan string) (*execplus.CmdPlus, error) {
-	cmdPlus := execplus.NewCmdPlus("docker-compose", "down")
+	cmdPlus := execplus.NewCmdPlus("docker-compose", "down", "--remove-orphans")
```

Comment:
Are we sure we want to do this? This could possibly remove containers on the user's machine that Exosphere doesn't manage. I know we were running into an issue with orphaned containers, is that something the new `exo clean` can fix? (Removing all containers listed in `docker-compose.yml`).